src/MoBi.Core/Services/IPKSimStarter.cs

```diff
@@ -1,8 +1,13 @@
-﻿namespace MoBi.Core.Services
+﻿using OSPSuite.Core.Domain.Builder;
+
+namespace MoBi.Core.Services
 {
    public interface IPKSimStarter
    {
       void StartPopulationSimulationWithSimulationFile(string simulationFilePath);
       void StartWithWorkingJournalFile(string journalFilePath);
+      ExpressionProfileBuildingBlock CreateMetabolizingEnzymeExpression();
```

Comment:
I am wondering why these are 3 methods and not one with a parameter since we already have an enum etc?